### ISSUE TITLE
I have a problem with IOU: 0.000000, iou_loss = 0.000000 for the third row while training p5 and p6

### ISSUE THREAD

**Issue by yuexiacangkuang** [AlexeyAB/darknet#8371]
When I train p5, IOU will appear in the third row of every four rows: 0.000000 iou_loss = 0.000000, as shown in Figure 1
![Figure1](https://user-images.githubusercontent.com/71477335/154774432-0a0e5067-8b02-4456-955b-d47c77fedd7f.png)
This will cause the average loss to not decrease, and the detection accuracy will be lower than that of yolov4 as shown in Figure 2.
![Figure2](https://user-images.githubusercontent.com/71477335/154774712-c912fd0c-4d65-4af3-8f5e-e3e0e60f0faa.png)
This problem also exists when training p6. The difference from before is that this time 0.000000 appears in the fourth row, that is to say, when training p5 and p6, 0.000000 appears in the last row each time.
![Figure3](https://user-images.githubusercontent.com/71477335/154774954-48fbb30d-d00f-4bc8-ab73-fc32359c58fa.png)
But this doesn't happen when I train yolov4.
![Figure4](https://user-images.githubusercontent.com/71477335/154775041-0ca7fa82-f2a7-4a1d-bb9a-7ecca4b30213.png)
The training situation of yolov4 is shown in the figure, and the average loss is very low.
![Figure5](https://user-images.githubusercontent.com/71477335/154775302-9b727409-7166-4cfa-90b3-f57710ac9e04.png)
Now I wonder why 0.000000 appears when training p5 p6, and 0.000000 might lead to low test accuracy.

**Comment by Ultiva** [comment]
Were you able to solve the issue ? 
For me it is currently the same (using p6 for a single-class)

**Comment by yuexiacangkuang** [comment]
> 
Not yet.

**Comment by Ultiva** [comment]
For me it works now using 
yolov4-p6.conv.289 weights instead of
yolov4-p6.weights, what I initially use.
